browser/src/control/Control.Zotero.js

```diff
@@ -736,7 +822,35 @@ L.Control.Zotero = L.Control.extend({
 				that.map.uiManager.showSnackbar(_('Failed to load collections'));
 			});
 
-		this.showItemsForUrl('https://api.zotero.org/users/' + this.userID + '/items/top' + this.getZoteroItemQuery(), true);
+			this.totalItems = -1;
+			this.totalPages = -1;
+			this._currentPage = 0;
+			fetch('https://api.zotero.org/users/'+ this.userID +'/items/top?v=3&key=' + this.apiKey + '&limit=1') //find total number of entries
+			.then(function (response) {
+				that.map.uiManager.showProgressBar(_("Loading references"),);
+				that.showItemsForUrl('https://api.zotero.org/users/' + that.userID + '/items/top' + that.getZoteroItemQuery(15, 0), true);
+
+				that.totalItems = parseInt(response.headers.get('Total-Results'));
+				that.totalPages = Math.ceil(that.totalItems/15);
```

Comment:
can we use variable for that `15` which is used in many places? so if we want to change number of rows it will be single place